File-Formats/Presentation/Working-with-find-and-replace.md

```diff
@@ -0,0 +1,932 @@
+//Creates a storage file from FileSavePicker
+StorageFile storageFile = await savePicker.PickSaveFileAsync();
+//Saves changes to the specified storage file
+await pptxDoc.SaveAsync(storageFile);
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+//Opens an existing presentation.           
+FileStream inputStream = new FileStream("Sample.pptx", FileMode.Open);
+using (IPresentation pptxDoc = Presentation.Open(inputStream))
+{
+	//Finds all the occurrences of a particular text in the PowerPoint presentation
+	ITextSelection[] textSelections = pptxDoc.FindAll("product", false, false);
+	foreach (ITextSelection textSelection in textSelections)
+	{
+		//Gets the found text containing text parts
+		foreach (ITextPart textPart in textSelection.GetTextParts())
+		{
+			//Sets highlight color
+			textPart.Font.HighlightColor = ColorObject.Yellow;
+		}
+	}
+	//Save the PowerPoint Presentation as stream
+	FileStream outputStream = new FileStream("Output.pptx", FileMode.Create);
+	pptxDoc.Save(outputStream);
+	//Release all resources of the stream
+	outputStream.Dispose();
+	//Closes the Presentation instance
+	pptxDoc.Close();
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//"App" is the class of Portable project.
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+Stream inputStream = assembly.GetManifestResourceStream("SampleBrowser.Presentation.Samples.Template.Sample.pptx");
+//Loads or open an PowerPoint Presentation
+IPresentation pptxDoc = Presentation.Open(inputStream);
+//Finds all the occurrences of a particular text in the PowerPoint presentation
+ITextSelection[] textSelections = pptxDoc.FindAll("product", false, false);
+foreach (ITextSelection textSelection in textSelections)
+{
+	//Gets the found text containing text parts
+	foreach (ITextPart textPart in textSelection.GetTextParts())
+	{
+		//Sets highlight color
+		textPart.Font.HighlightColor = ColorObject.Yellow;
+	}
+}
+//Create new memory stream to save Presentation.
+MemoryStream stream = new MemoryStream();
+//Save Presentation in stream format.
+pptxDoc.Save(stream);
+//Close the presentation
+pptxDoc.Close();
+stream.Position = 0;
+//The operation in Save under Xamarin varies between Windows Phone, Android and iOS platforms. Please refer presentation/xamarin section for respective code samples.
+if (Device.OS == TargetPlatform.WinPhone || Device.OS == TargetPlatform.Windows)
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+## Find and highlight in specific slide
+
+You can find all the occurences of a text in specific PowerPoint slide (slide, notes slide, master slide or layout slide) and highlight the found text.
```

Comment:
You can find all the occurrences of a text in a specific PowerPoint slide (slide, notes slide, master slide, or layout slide) and highlight the found text.

Reply:
Modified as you said.